### ISSUE TITLE
Adding support for Aspire environment variables

### ISSUE THREAD

**Issue by snakefoot** [juliuskoval/NLog.Targets.OpenTelemetryProtocol#17]
Maybe setup OpenTelemetry-Target to recognize these Environment-Variables by default unless nothing else as been assigned:
- OTEL_EXPORTER_OTLP_ENDPOINT
- OTEL_EXPORTER_OTLP_PROTOCOL
- OTEL_EXPORTER_OTLP_HEADERS
- OTEL_RESOURCE_ATTRIBUTES

Where OTEL_EXPORTER_OTLP_ENDPOINT maps to Layout-property:
- Endpoint = `"${environment:OTEL_EXPORTER_OTLP_ENDPOINT}"`

Where OTEL_EXPORTER_OTLP_PROTOCOL can have following values:
- "http/protobuf" => OtlpProtocol.HttpProtobuf,
- "grpc" => OtlpProtocol.Grpc,
- _ => options.Protocol

Where OTEL_EXPORTER_OTLP_HEADERS + OTEL_RESOURCE_ATTRIBUTES must be parsed as comma-seperated mapping (`key1=value1;key2=value2`):
```
        foreach (var part in config?.Split(',') ?? [])
        {
            if (part.Split('=') is { Length: 2 } parts)
                resourceAttributes.Add(parts[0], parts[1]);
        }
```

**Comment by richbeales** [comment]
This would also prevent the need to have our access token/key committed to source control (if we can pick up the OTLP headers from an env variable)

**Comment by juliuskoval** [comment]
I tested the latest version of the package and the environment variables were respected if I left the appropriate field empty in nlog.config, otherwise the value from the config was used. The only problem was that the environment variable for resources didn't work if I set UseDefaultResources to false, so I'll fix that.

@richbeales Could you test this?

**Comment by juliuskoval** [comment]
Actually, it looks like environment variables fall under default resources, so it looks like UseDefaultResources=false means they should be ignored.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/d8ce51b1b482e7432da0e7498fdc1052eb2ca807/src/OpenTelemetry/Resources/ResourceBuilder.cs#L57

**Comment by snakefoot** [comment]
Yes looks like it will automatically apply headers etc unless specified by NLog target:

- https://github.com/open-telemetry/opentelemetry-dotnet/blob/da905a8264cd9383ff80df1bfbce21f0a48376fc/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs#L239

Then I think my worries was wrong. Just need to update the documentation about environment-variables will be used by default, when not having specified anything (And using default value `UseDefaultResources=true`).

**Comment by snakefoot** [comment]
Created #18 to close this issue
